src/problems/MetalAdvectionProblem/test_sne.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef TEST_SNE_ // NOLINT
+#define TEST_SNE_
+//==============================================================================
+// TwoMomentRad - a radiation transport library for patch-based AMR codes
+// Copyright 2020 Benjamin Wibking.
+// Released under the MIT license. See LICENSE file included in the GitHub repo.
+//==============================================================================
+/// \file test_hydro3d_blast.hpp
+/// \brief Defines a test problem for a 3D explosion.
+///
+
+// external headers
+#include <fmt/format.h>
+#include <fstream>
+
+constexpr double Msun = 2.e33;
+constexpr double Const_G = 6.67e-8;
+constexpr double yr_to_s = 3.154e7;
+constexpr double Myr = 1.e6 * yr_to_s;
+constexpr double pc = 3.018e18;
+constexpr double kpc = 1.e3 * pc;
+constexpr double Mu = 0.6;
+constexpr double kmps = 1.e5;
+constexpr double Const_mH = 1.67e-24;
+constexpr double kb = 1.3807e-16;
+constexpr double sqrtpi = 1.772453;
+
```

Comment:
Some of these constants can be replaced with the versions in `fundamental_constants.H` in Microphysics. Since those are already used elsewhere in Quokka, it would be good to do this for consistency throughout the code.

Can you add a `// TODO: replace with Microphysics constants` here so someone can do this in the future?